pkg/controller/services.go

```diff
@@ -924,6 +925,11 @@ func (cont *AciController) opflexDeviceChanged(obj apicapi.ApicObject) {
 	domName := obj.GetAttrStr("domName")
 	ctrlrName := obj.GetAttrStr("ctrlrName")
 
+	name, err := os.Hostname()
```

Comment:
We can call this "controllerPodHostName" maybe?

Reply:
sure, that's more helpful. Thanks